### PR TITLE
Master

### PR DESCRIPTION
**Pull request by jeremyckahn**
Aye, I don't think there's a way to make `master` the "gh-pages branch," per #21.  So I'll just make this manual PR.  Unless you know another way, @dancrumb?

### CODE REVIEW

File: Gruntfile.js [hunk 1]
```diff
@@ -42,20 +19,10 @@ module.exports = function (grunt) {
         logo: 'http://cdn.jellyvision.com/wp-content/themes/jellyvision/img/header-logo.png',
         options: {
           paths: ['scripts'],
-          outdir: 'dist/doc'
+          outdir: 'doc'
```
Comment (dancrumb) [inline Gruntfile.js:22]: As discussed... change to `docs`

File: doc/assets/js/api-list.js [hunk 2]
```diff
@@ -97,15 +98,17 @@ tabview.get('panelNode').all('a').each(function (link) {
 
 // -- Private Functions --------------------------------------------------------
 function getFilterResultNode() {
-    return filter.get('queryType') === 'classes' ? classesNode : modulesNode;
+    var queryType = filter.get('queryType');
```
Comment (dancrumb) [inline doc/assets/js/api-list.js:101]: FWIW, there's some nice typesetting you can use for these nested ternaries:

```
return  queryType === 'classes'    ? classesNode
      : queryType === 'elements'   ? elementsNode 
      :                              modulesNode;
```

**Review by dancrumb — APPROVED** [review]

File: doc/assets/js/api-list.js (hunk 2 above)
Comment (jeremyckahn) [reply doc/assets/js/api-list.js:101]: This is third-party (YUIDoc) code that, for better or worse, is tracked at the moment by Git.  But I do prefer your version!

File: Gruntfile.js (hunk 1 above)
Comment (jeremyckahn) [reply Gruntfile.js:22]: Done in #25.

**Pull request closed without merging**